packages/website/tools/generate-website-dts.ts

```diff
@@ -52,10 +52,10 @@ function replaceImports(text: string, from: string, to: string): string {
 function injectImports(text: string, from: string, safeName: string): string {
   const regex = new RegExp(`import\\(["']${from}["']\\)`, 'g');
   if (regex.test(text)) {
-    return (
-      `import type * as ${safeName} from '${from}';\n` +
-      text.replace(regex, safeName)
-    );
+    return `import type * as ${safeName} from '${from}';\n${text.replace(
```

Comment:
Make `\n` a literal character

Reply:
This is just a curiosity question — do you feel it's a case-by case decision on whether to use an explicit `\n` vs a literal newline?

I was just curious because there are hundreds of places throughout our codebases that use explicit `\n`s, so I was wondering if you were in favor of any consistent pattern, or if it just depends?